src/YesSql.Core/Commands/IndexCommand.cs

```diff
@@ -67,13 +67,19 @@ protected static void GetProperties(DbCommand command, object item, string suffi
 
         protected static PropertyInfo[] TypePropertiesCache(Type type)
         {
-            if (TypeProperties.TryGetValue(type.FullName, out var pis))
+            if (TypeProperties.TryGetValue(type, out var pis))
             {
                 return pis;
             }
+            //Avoid InvalidCastException
```

Comment:
New line before the comment. Single space after the //

Reply:
done